src/linear_algebra/basic.lean

```diff
@@ -579,6 +579,7 @@ variables {R}
 instance [subsingleton M] : unique (submodule R M) :=
 ⟨⟨⊥⟩, λ a, @subsingleton.elim _ ((subsingleton_iff R).mp ‹_›) a _⟩
 
+instance [nontrivial M] : nontrivial (submodule R M) := nontrivial_iff.mp ‹_›
 lemma bot_ne_top [nontrivial M] : (⊥ : submodule R M) ≠ ⊤ :=
```

Comment:
Is this lemma maybe subsumed by `_root_.bot_ne_top` now?

Reply:
Oh, good thought. It probably is, although I need to wait for a cache before I can find out locally.